src/components/pages/search.js

```diff
@@ -152,11 +152,14 @@ function Search({ PageTemplate }) {
     }
   ];
 
+  const showSearchingHeadline = isSearching && documents.length === 0;
```

Comment:
Added the `documents.length === 0` restriction as well so that we don't see the headline briefly changing each time the search text changes